src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/CommentTabPanel.kt

```diff
@@ -17,14 +17,15 @@ class CommentTabPanel(
 
     private val loadMoreCommentsJira8Locator: By = By.cssSelector("a.show-more-comments")
     private val loadMoreCommentsJira9Locator: By = By.cssSelector("button.show-more-comment-tabpanel")
+    private val duration: Duration = Duration.ofMinutes(2)
```

Comment:
Would a use actually wait 2 minutes?

Reply:
If they want to see all comments then they have to wait several minutes, in some cases much more than 2, but I know  that usually users give up after few seconds. If I change the value to more realistic we would important information about how our product behave under high load.

Reply:
I think that we can assume that the VU who is doing the performance measures for us is rather patient for the purpose of finishing the action even if it's slow. The VU should be still have limited patience, but it's purpose is a bit different then the regular user patience, which is to not get stuck.

I'm fine with high patience of the automation as long as we have processes to monitor it (and the test does that).

Reply:
Reasonable justification :+1: 
Let's share it via a code comment to make it obvious to everyone.